sw/device/tests/BUILD

```diff
@@ -2051,14 +2053,53 @@ opentitan_functest(
     ],
 )
 
+otp_json(
+    name = "power_virus_systemtest_otp_overlay",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                "CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG": "0000090606",
```

Comment:
We should probably update the documentation in `hw/ip/otp_ctrl/data/otp_ctrl_img_creator_sw_cfg.hjson` to say that this OTP word is byte-packed Mubi4 values.

Reply:
fixed.